src/components/InternalDropdown/pageObject/index.js

```diff
@@ -75,6 +84,73 @@ class PageInternalDropdown {
             bottom: y + height,
         };
     }
+
+    /**
+     * Returns true when the arrow to scroll down exits, false otherwise.
+     * @method
+     * @returns {bool}
+     */
+    arrowDownExists() {
+        return $(this.rootElement)
+            .$('[data-id="internal-dropdown-arrow-down"]')
+            .isExisting();
+    }
+
+    /**
+     * Returns true when the arrow to scroll down exits, false otherwise.
+     * @method
+     * @returns {bool}
+     */
+    arrowUpExists() {
+        return $(this.rootElement)
+            .$('[data-id="internal-dropdown-arrow-up"]')
+            .isExisting();
+    }
+    /**
+     * Returns true when the search no results found, false otherwise.
+     * @method
+     * @returns {bool}
+     */
+    emptyMessageExist() {
+        return $(this.rootElement)
+            .$('p*=Our robots')
```

Comment:
What happen if we decide to change this text? We should use a unique selector that will not change.